docs/README.md

```diff
@@ -168,6 +144,102 @@ That's all you need to do. The `versions` directory is listed on server start to
+  - make sure to [install `direnv`](https://direnv.net/docs/installation.html) and run `direnv allow` at the root of the **expo/** repo.
+- have gone through the steps mentioned in [**"Download and Setup" in the contribution guideline**](https://github.com/expo/expo/blob/main/CONTRIBUTING.md#-download-and-setup).
+- can run **expo/docs** repo **[locally](https://github.com/expo/expo/tree/main/docs#running-locally)**.
+
+Once you have made sure the development setup is ready, proceed to the next section:
+
+#### Step 1: Update the package’s TypeDoc
+
+- After you have identified which package docs you want to update, open a terminal window and navigate to that package’s directory. For example:
+
+```shell
+# Navigate to expo-constants package directory inside expo/ repo
+cd expo/packages/expo-constants
+```
+
+- Then, open **.ts** file in your code editor/IDE where you want to make changes/updates.
+- Start the TypeScript build compilation in watch mode using `yarn build` in the terminal window.
+- Make the update. For example, we want to update the TypeDoc description of [`manifest2` property](https://docs.expo.dev/versions/latest/sdk/constants/#nativeconstants)
+
+  - Inside the **src/** directory, open **Constants.types.ts** file.
+  - Search for `manifest2` property. It has a current description as shown below:
+
+  ```ts
+  /**
+   * New manifest for Expo apps using modern Expo Updates from a remote source, such as apps that
+   * use EAS Update. Returns `null` in bare workflow and when `manifest` is non-null.
+   * > Prefer using `Constants.expoConfig` instead, which behaves more consistently across classic
+   * updates and modern Expo Updates.
+   */
+  manifest2: Manifest | null;
+  ```
+
+- In the above example, let’s remove the word "New" and update some content:
+
+```ts
+/**
+ * Manifest for Expo apps using modern Expo Updates from a remote source, such as apps that
+ * use EAS Update. Returns `null` in bare workflow and when `manifest` is non-null.
+ * > Use `Constants.expoConfig` instead, which behaves more consistently across classic
+ * updates and modern Expo Updates.
+ */
+manifest2: Manifest | null;
+```
+
+- Before moving to the next step, make sure to exit the "watch mode" by pressing `Ctrl + C` from the keyboard.
+
+#### Step 2: Apply TypeDoc updates to expo/docs repo
+
+In the terminal window and run the following command with `expotools` (`et`) to generate the JSON data file for the package (which is stored at the location `expo/docs/public/static/data/[SDK-VERSION]`)
+
+- Read the **NOTE** in the below snippet for updating the docs for `unversioned`:
+
+```shell
+# et short of expotools
+# gdad is short for generate-docs-api-data
+# -p is short for --package
+# -s is short for --sdk
+
+et gdad -p expo-constants -s 47
+
+#### NOTE ####
+# To update unversioned docs, run the command without mentioning the SDK version
+et gdad -p expo-constants
+```
```

Comment:
Maybe we can skip explaining everything which this commands do, and just recommend to run `et gdad --help` for more information?

Reply:
We could also use the long version of the command, so we don't have to explain the abbreviations:

`et generate-docs-api-data --packageName expo-constants --sdk 47`
(for more information about this command, use `et generate-docs-api-data --help`)